code/game/atoms.dm

```diff
@@ -1789,6 +1789,11 @@
  *
  * micro-optimized to hell because this proc is very hot, being called several times per movement every movement.
  *
+ * HEY JACKASS, LISTEN
+ * IF YOU ADD SOMETHING TO THIS PROC, MAKE SURE /mob/living ACCOUNTS FOR IT
```

Comment:
is this unit testable

Reply:
not really. I could test to ensure existing cases work, that might be good, but I can't really test for potential problem sources since I'd uh. need to know they existed.